### PR TITLE
update ImageSummary to return the history of an image

### PR DESCRIPTION
**Pull request by alexstan12**
**What type of PR is this?**
feature

**Which issue does this PR fix**:
it adds information regarding command history of an image

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

Closing https://github.com/project-zot/zot/issues/748

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: examples/config-policy.json [hunk 1]
```diff
@@ -1,113 +1,115 @@
...
-        ]
-      }
-    }
-  },
-  "log": {
-    "level": "debug",
-    "output": "/tmp/zot.log"
-  }
-}
+{
```
Comment (andaaron) [inline examples/config-policy.json:1]: Revert this change.

File: .gitignore [hunk 2]
```diff
@@ -24,4 +24,5 @@ coverage.html
 tags
 vendor/
 .vscode/
-examples/config-sync-localhost.json
\ No newline at end of file
+examples/config-sync-localhost.json
+examples/config-policy-alexs.json
```
Comment (andaaron) [inline .gitignore:28]: Revert this.

File: pkg/extensions/search/resolver.go [hunk 3]
```diff
@@ -540,36 +562,124 @@ func (r *queryResolver) getImageList(store storage.ImageStore, imageName string)
...
+				layers,
+				layer,
+			)
+
+			missingHistoryField := "<missing>"
+
+			allHistory = append(allHistory, &gql_generated.LayerHistory{
+				Layer: layer,
+				HistoryDescription: &gql_generated.HistoryDescription{
+					Comment:   &missingHistoryField,
```
Comment (andaaron) [inline pkg/extensions/search/resolver.go:599]: Should be `nil` instead of an empty string.

**Review by shimish2 — APPROVED** [review]
lgtm

**Review by rchincha — APPROVED** [review]
lgtm
